### PR TITLE
Osc ops

### PR DESCRIPTION
**Pull request by HumairAK**
Resolve: https://github.com/operate-first/support/issues/640

### CODE REVIEW

**Comment by HumairAK** [comment]
Also ran the sigs.yaml generation tool, hence the other updates (separated to another commit)

**Review by harshad16 — APPROVED** [review]
lgtm
/approve

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -2,6 +2,9 @@ aliases:
   sig-community-leads:
     - quaid
     - schwesig
+  sig-data-science-leads:
+    - MichaelClifford
+    - unknown
```
[the remark this answers is not in the document]
Comment (HumairAK) [reply OWNERS_ALIASES:7]: what is unknown? @MichaelClifford  -- is this intended?

[the remark this answers is not in the document]
Comment (HumairAK) [reply OWNERS_ALIASES:7]: bump

**Pull request closed without merging**